docs/source/user-guide/tasks/manage-environments.rst

```diff
@@ -337,13 +337,35 @@ the environment, and running any activation scripts that the environment may
 contain. These activation scripts are how packages can set arbitrary
 environment variables that may be necessary for their operation.
 
+When `installing Anaconda <http://docs.continuum.io/anaconda/install.html>`_,
+you have the option to “Add Anaconda
+to my PATH environment variable.” This is not recommended because the
+add to PATH option appends Anaconda to path. This means Anaconda’s features
```

Comment:
Only kind of correct on Windows, but misses some important details (sorry I should've mentioned them already). On Windows, PATH is composed from two parts, the system PATH and the user PATH. The system PATH always comes first. When you install Anaconda for Just Me, we add it to the user PATH, when you install it for All Users, we add it to the system PATH. In the former case, you can end up with system PATH values taking precedence over our entries. In the latter case you do not (but we don't recommend All User installs).